sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -2829,41 +2824,6 @@ private RxDocumentServiceRequest getPermissionRequest(String userLink, Permissio
                 Utils.joinPath(Paths.OFFERS_PATH_SEGMENT, null));
     }
 
-    private <T extends Resource> Flux<FeedResponse<T>> readFeedCollectionChild(FeedOptions options, ResourceType resourceType,
```

Comment:
@mbhaskar isn't this method needed in future?

Reply:
If required, can we please comment it out, instead of removing it ?

Reply:
Ok will comment out

Reply:
done in the lastest [commit](https://github.com/Azure/azure-sdk-for-java/pull/9111/commits/b972e8c088bb80842e7fa0633740f49128f1991b).